projects/1.21/assets/creeperhost-presents-steves-carts/stevescarts/lang/zh_cn.json

```
@@ -0,0 +1,711 @@
+  "info.stevescarts.presentDuplicateError": "[%1] 不允许添加两次。",
+  "info.stevescarts.presentNemesisError": "[%1] 无法与 [%2] 一起运行",
+  "info.stevescarts.shiftForMore": "按下 <[%1]> 显示更多",
+  "info.stevescarts.sideClashError": "[%1] 和 [%2] 会在 [%3] 中冲突",
+  "info.stevescarts.sidesAnd": "以及",
+  "info.stevescarts.storageEmpty": "空白",
+  "info.stevescarts.storageFull": "满",
+  "info.stevescarts.toolDurability": "耐久：[%1]%",
+  "info.stevescarts.toolUnbreakable": "耐久",
+  "item.stevescarts.BlockDetector0": "探测管理器",
+  "item.stevescarts.BlockDetector1": "探测装置",
+  "item.stevescarts.BlockDetector2": "探测站",
+  "item.stevescarts.BlockDetector3": "探测结点",
+  "item.stevescarts.BlockDetector4": "探测红石装置",
+  "item.stevescarts.advanced_control_system": "高级控制系统",
+  "item.stevescarts.advanced_crafter": "高级合成器",
+  "item.stevescarts.advanced_pcb": "高级印刷电路板",
+  "item.stevescarts.advanced_sctank": "高级储罐",
+  "item.stevescarts.advanced_shooter": "高级发射器",
+  "item.stevescarts.advanced_smelter": "高级熔炉",
+  "item.stevescarts.advanced_solar_panel": "高级太阳能板",
+  "item.stevescarts.advanced_thermal_engine": "高级热能引擎",
+  "item.stevescarts.basic_drill": "基础钻头",
+  "item.stevescarts.basic_farmer": "基础种植器",
+  "item.stevescarts.basic_solar_engine": "基础太阳能引擎",
+  "item.stevescarts.basic_wood_cutter": "基础伐木器",
+  "item.stevescarts.basket": "篮子",
+  "item.stevescarts.birch_log": "白桦原木",
+  "item.stevescarts.birch_twig": "白桦树枝",
+  "item.stevescarts.blade_arm": "刀片臂",
+  "item.stevescarts.blank_upgrade": "空白升级",
+  "item.stevescarts.blockactivator": "模块开关管理器",
+  "item.stevescarts.blockadvdetector": "高级探测轨道",
+  "item.stevescarts.blockcargomanager": "货物管理器",
+  "item.stevescarts.blockcartassembler": "矿车装配机",
+  "item.stevescarts.blockdistributor": "外部分配机",
+  "item.stevescarts.blockjunction": "交接轨道",
+  "item.stevescarts.blockliquidmanager": "流体管理器",
+  "item.stevescarts.blue_pigment": "蓝色颜料",
+  "item.stevescarts.brake_handle": "制动手柄",
+  "item.stevescarts.bridge_builder": "架桥器",
+  "item.stevescarts.burning_easter_egg": "燃烧复活节彩蛋",
+  "item.stevescarts.cage": "兽笼",
+  "item.stevescarts.cake_server": "蛋糕服务器",
+  "item.stevescarts.chest_lock": "箱子锁",
+  "item.stevescarts.chest_pane": "箱子面板",
+  "item.stevescarts.chocolate_easter_egg": "巧克力复活节彩蛋",
+  "item.stevescarts.chunk_loader": "区块加载器",
+  "item.stevescarts.cleaning_core": "清理核心",
+  "item.stevescarts.cleaning_fan": "清理风扇",
+  "item.stevescarts.cleaning_machine": "清理器",
+  "item.stevescarts.cleaning_tube": "清理管道",
+  "item.stevescarts.coal_engine": "煤炭引擎",
+  "item.stevescarts.color_randomizer": "颜色随机器",
+  "item.stevescarts.colorizer": "着色器",
+  "item.stevescarts.compact_solar_engine": "紧凑型太阳能引擎",
+  "item.stevescarts.crafter": "合成器",
+  "item.stevescarts.creative_engine": "创造模式引擎",
+  "item.stevescarts.creative_hull": "创造模式外壳",
+  "item.stevescarts.creative_incinerator": "创造模式焚化炉",
+  "item.stevescarts.creative_sctank": "创造模式储罐",
+  "item.stevescarts.creative_supplies": "创造模式供应",
```

有些怪怪的，应该是消耗品的无限补给
```suggestion
  "item.stevescarts.creative_supplies": "创造模式物资供应",
```
仅建议，改成这样也怪怪的